src/crypto/shelley/transactions/utils.js

```diff
@@ -1,10 +1,22 @@
 // @flow
 
-import {InputOutput, PublicKey, PrivateKey} from 'react-native-chain-libs'
-import {HdWallet} from 'react-native-cardano'
+import {
+  AccountBindingSignature,
+  Certificate,
+  InputOutput,
+  PayloadAuthData,
+  StakeDelegation,
+  StakeDelegationAuthData,
+} from 'react-native-chain-libs'
+import type {BaseSignRequest} from '../../../types/HistoryTransaction'
+import {CONFIG} from '../../../config'
+
 import {BigNumber} from 'bignumber.js'
 
-export const getTxInputTotal = async (IOs: InputOutput): Promise<BigNumber> => {
+export const getTxInputTotal = async (
+  IOs: InputOutput,
+  shift: boolean,
```

Comment:
I’m not in love with the naming here. I would make it more obvious as passing an enum to specify `ADA` or `Lovelace` the shift is too low level